src/overlays/actors/ovl_En_GirlA/z_en_girla.c

```diff
@@ -1,14 +1,68 @@
 #include "z_en_girla.h"
+#include "vt.h"
 
 #define FLAGS 0x00000019
 
 #define THIS ((EnGirlA*)thisx)
 
-void EnGirlA_Init(Actor* thisx, GlobalContext* globalCtx);
-void EnGirlA_Destroy(Actor* thisx, GlobalContext* globalCtx);
-void EnGirlA_Update(Actor* thisx, GlobalContext* globalCtx);
+void EnGirlA_Init(Actor* this, GlobalContext* globalCtx);
+void EnGirlA_Destroy(Actor* this, GlobalContext* globalCtx);
+void EnGirlA_Update(Actor* this, GlobalContext* globalCtx);
+
+void func_80A3BEAC(GlobalContext*, EnGirlA*);
```

Comment:
Ideally, forward declarations like these should have argument names (ie they can just be a copy paste of the actual function header)